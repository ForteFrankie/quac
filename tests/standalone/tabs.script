#!/bin/bash

# Find source files that contain tabs. There should be none.
#
# Specifically, for each file containing at least one tab, this test prints
# the number of lines containing a tab. Finding these lines is up to you. :)
# (We don't print line numbers because that will change based on non-tab
# revisions.)

# Copyright (c) 2012-2013 Los Alamos National Security, LLC, and others.

. ./environment.sh

# don't use $QUACBASE because it's an absolute path
cd ../..

# see <http://stackoverflow.com/a/4767577/396038>
find . -name .git -prune -o -type f -print0 | xargs -0 file | egrep 'Python|shell|C source' | cut -d: -f1 | xargs fgrep -Hc '	' | egrep -v ':0$'
