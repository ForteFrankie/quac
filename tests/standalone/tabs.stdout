./tests/standalone/tabs.script:1
./tests/standalone/hashsplit.script:22
./tests/standalone/quacreduce.script:1
./bin/tsvgrep:1
./lib/u.py:1
